README.md

```diff
@@ -12,6 +12,7 @@ The first twenty-four examples are being added as an advent calendar leading up
 
 1. [Creating Tables](recipes/01/01_create_table.md)
 2. [Filtering Data](recipes/02/02_where.md)
+3. [Aggregating Data](recipes/03/03_group_by.md)
```

Comment:
I was wondering whether we should start a new section for this called "Aggregations". We can also split this up later, though.

Reply:
Maybe we can wait until we put out some more examples to see what grouping (😬 ) would make sense?

Reply:
:+1: